projects/frontend/shared-components/gui/projects/shared/src/lib/commons/css/utils.scss

```diff
@@ -0,0 +1,34 @@
+
+/**
+ * Theme-aware CSS utility
+ * Usage: Invoke mixin, including custom theme-specific CSS properties, inside braces `{ ... }`.
+ * Example:
+ *
+ * ```
+ *   .page-example-image {
+ *       width: 100px;
+ *       height: 100px;
+ *
+ *       @include light-theme {
+ *           background-image: url('icon-light.png');
+ *           border-color: black;
+ *       }
+ *       @include dark-theme {
+ *           background-image: url('icon-dark.png');
+ *           border-color: white;
+ *       }
+ *    }
+ * ```
+ */
+@mixin light-theme {
+    :host-context(body:not([cds-theme])) &,
```

Comment:
nit: maybe add ticket to resolve those warnings

Reply:
Not sure what this is. 😅 Are you referring to the static code analysis warnings?

Reply:
Yeah - modifying the code opened in order to handle the quality warnings, is certainly out of scope. 
Could we link to the warnings in a ticket, so we revise those in an additional effort planned?